src/Components/SignUpForm/SignUpForm.module.scss

```diff
@@ -0,0 +1,156 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap");
+
+@mixin flex-display($direction, $content) {
+  display: flex;
+  flex-direction: $direction;
+  justify-content: $content;
+  align-items: center;
+}
+
+.formContainer {
+  width: 32vw;
+  margin: 0 auto;
+  background: #f8f8f8;
+  border: 1px solid #b8b8b8;
+  border-radius: 2rem;
+  padding: 1rem;
+
+  .formHeader {
+    @include flex-display(column, center);
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1.5rem;
+    line-height: 2.3rem;
+    letter-spacing: 0.02em;
+    margin: 2rem auto;
+    color: #578cf1;
+  }
+
+  .SignupForm {
+    @include flex-display(column, center);
+    margin-bottom: 2rem;
+
+    label {
+      width: 74%;
+    }
+
+    .formInput {
+      width: 100%;
+      margin: 0.3rem auto;
+      padding: 2% 3%;
+      border: 0.5px solid #b3b3b3;
+      border-radius: 6px;
+      background: #f8f8f8;
+
+      &:focus {
+        outline: none;
+      }
+
+      &::placeholder {
+        font-family: "Poppins";
```

Comment:
Use the variable defined in index.scss. Always use variables. Check for all the scss files you have written